app/pandas.ipynb

```diff
@@ -208,18 +475,88 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "df[\"Average age\"] = df[\"Age\"].mean()\n",
+    "df"
```

Comment:
Adding 'Average age' as a column with the same value for all rows is not typically useful. Consider calculating it separately and not adding it as a column unless specifically required.